libs/ui/src/lib/timed-notice/timed-notice.component.html

```diff
@@ -0,0 +1,54 @@
+<div
+  class="cvi-timed-notice__container cvi-timed-notice__container--severity-{{
+    severity
+  }}"
+>
+  <div class="cvi-timed-notice__icon-container">
+    <cvi-ng-icon [name]="getIconName()" [height]="24"></cvi-ng-icon>
+  </div>
+  <div class="cvi-timed-notice__content">
```

Comment:
@helemand I wasn't able to commit to your fork, but here're suggested changes on how we can get rid of the classes outside of the component – we could replace the "aside" content projection slot in `TimedNoticeComponent` (Angular) with `asideItems` prop. 

[timed-notice-updates.zip](https://github.com/e-gov/cvi/files/12330567/timed-notice-updates.zip)